src/main.rs

```diff
@@ -162,32 +162,38 @@ enum Command {
         #[structopt(short, long, parse(from_os_str))]
         target_dir: Option<PathBuf>,
     },
-    /// Compiles the smart contract
+    /// Compiles the contract, generates metadata, bundles both together in a '.contract' file
     #[structopt(name = "build")]
     Build {
         /// Path to the Cargo.toml of the contract to build
         #[structopt(long, parse(from_os_str))]
         manifest_path: Option<PathBuf>,
+        /// Only the Wasm and the metadata are generated, no bundled .contract file is created
+        #[structopt(long = "skip-bundle", conflicts_with = "skip-metadata")]
+        skip_bundle: bool,
+        /// Only the Wasm is created, generation of metadata and a bundled .contract file is skipped
+        #[structopt(long = "skip-metadata", conflicts_with = "skip-bundle")]
+        skip_metadata: bool,
         #[structopt(flatten)]
         verbosity: VerbosityFlags,
         #[structopt(flatten)]
         unstable_options: UnstableOptions,
     },
-    /// Compiles the contract, generates metadata, packs both together in one file
-    #[structopt(name = "pack")]
-    Pack {
-        /// Path to the Cargo.toml of the contract to build and pack
+    /// Command has been deprecated, use 'cargo contract build' instead
+    #[structopt(name = "generate-metadata")]
+    GenerateMetadata {
+        /// Path to the Cargo.toml of the contract to build
```

Comment:
This command variant can be empty now since none of the fields are used for just the deprecation warning